darts/datasets/__init__.py

```diff
@@ -0,0 +1,77 @@
+from .dataset_loaders import DatasetLoaderCSV, DatasetLoaderMetadata
+
+"""
+    Overall usage of this package:
+    from darts.datasets import AirPassengers
+    ts: TimeSeries = AirPassengers.load()
+"""
+AirPassengersDataset = DatasetLoaderCSV(
+    metadata=DatasetLoaderMetadata(
+        "air_passengers.csv",
+        uri="https://raw.githubusercontent.com/unit8co/darts/master/examples/AirPassengers.csv",
```

Comment:
instead of leaving the CSV files in `examples/` could we put them in a (new) `datasets/` folder?

Reply:
the goal is to move them (new branch, new repo, cloud hosting, cdn, whatever) indeed. for now we're using github for PoC.